### PR TITLE
markdown: document this function

### PR DESCRIPTION
**Pull request by jordigh**

### CODE REVIEW

**Comment by paulfitz** [comment]
(turning on a preview just to show recent changes to someone, sorry for noise)

File: app/client/lib/markdown.ts [hunk 1]
```diff
@@ -2,6 +2,22 @@ import { sanitizeHTML } from 'app/client/ui/sanitizeHTML';
 import { BindableValue, DomElementMethod, subscribeElem } from 'grainjs';
 import { marked } from 'marked';
 
+/**Helper function for using Markdown in grainjs elements. It accepts
```
Comment (berhalak) [inline app/client/lib/markdown.ts:5]:
```suggestion
/**
 * Helper function for using Markdown in grainjs elements. It accepts
```

Comment (berhalak) [reply app/client/lib/markdown.ts:5]: Indentation

**Review by berhalak — CHANGES_REQUESTED** [review]
Looks good.

File: app/client/lib/markdown.ts [hunk 2]
```diff
@@ -2,6 +2,24 @@ import { sanitizeHTML } from 'app/client/ui/sanitizeHTML';
 import { BindableValue, DomElementMethod, subscribeElem } from 'grainjs';
 import { marked } from 'marked';
 
+/**
+ * Helper function for using Markdown in grainjs elements. It accepts
+ * both plain Markdown strings, as well as methods that use an observable.
+ * Example usage:
+ *
+ *    cssSection(markdown(t(`# New Markdown Function
+ *
```
Comment (berhalak) [inline app/client/lib/markdown.ts:11]: Empty line

**Review by berhalak — APPROVED** [review]
Thanks.

File: app/client/lib/markdown.ts (hunk 2 above)
Comment (jordigh) [reply app/client/lib/markdown.ts:11]: That's on purpose, to demonstrate new paragraphs in markdown.

**Pull request merged**